R/model.R

```diff
@@ -146,11 +147,11 @@ run_metapop_simulation <- function(
   if (nrow(mixing) != length(parameters)) {
     stop('mixing matrix rows must match length of parameters')
   }
-  if (!all(round(rowSums(mixing), 1) == 1)) {
+  if (!all(lapply(1:length(parameters), function(x){approx_sum(mixing[x,], 1)}) == T)) {
     stop('all mixing matrix rows must sum to 1')
   }
-  if (!all(round(colSums(mixing), 1) == 1)) {
-    stop('all mixing matrix columns must sum to 1')
+  if (!all(lapply(1:length(parameters), function(x){approx_sum(mixing[,x], 1)}) == T)) {
+    print('warning: mixing matrix is asymmetrical')
```

Comment:
if we use `warning` then people can supress those outputs if they're annoying with `suppressWarnings`

```suggestion
    warning('mixing matrix is asymmetrical')
```